src/clients/apiClient.ts

```diff
@@ -9,4 +10,11 @@ export const checkBackendConnection = async (): Promise<boolean> => {
   }
   
   return response.ok;
+
+
+}
+
+export async function fetchSpeciesQuery(tailType: number, size: number, colour: string ) {
```

Comment:
This function should return a promise as we are getting a list of species
```suggestion
export async function fetchSpeciesQuery(tailType: number, size: number, colour: string ): Promise<Response> {
```

Reply:
We want to create a new interface to pass into this method, eg.
```suggestion
export interface SpeciesSearch {
    tailType: number;
    size: number;
    colour: string;
}

export async function fetchSpeciesQuery(speciesSearch: SpeciesSearch) {
```

You would then access the properties in the method using `speciesSearch.tailType`